libcontainer/selinux/selinux.go

```diff
@@ -58,16 +57,30 @@ func getSelinuxMountPoint() string {
 	}
 	selinuxfs = ""
 
-	mounts, err := mount.GetMounts()
+	f, err := os.Open("/proc/self/mountinfo")
 	if err != nil {
 		return selinuxfs
 	}
-	for _, mount := range mounts {
-		if mount.Fstype == "selinuxfs" {
-			selinuxfs = mount.Mountpoint
-			break
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		txt := scanner.Text()
+		sepIdx := strings.Index(txt, " - ")
```

Comment:
I realise this is exactly what the old code did, but Is this field always guaranteed to be `-`? Why not just look at the 9th field? Or is this to protect against spaces in the mountpoint names?